src/internal/validators.py

```diff
@@ -34,8 +34,8 @@ def validate_semester_float(value):
         )
 
 
-discord_username_regex = re.compile(r'^(.+)#([0-9]{4})$')
+discord_username_regex = re.compile(r'^(?!.*\.\.)[a-z0-9_.]{2,32}$')
```

Comment:
To repeat the sentiment of [this comment](https://github.com/MAKENTNU/web/blob/70270b1eb6e69f2d2dfd99e84d179f17a6483e34/src/internal/models.py#L58), I don't think we should write code that keeps track of these minute details of other services as long as it doesn't affect us much 🤔 The original regex was mainly there to help users input the correct value, in case they forgot to include the four digits at the end (which I think used to be a 
fairly common problem), and because it was such a simple regex string (notice in particular the previous `(.+)`, which matched the whole username except the hashtag and last four digits). However, I think the new regex - which I'm guessing is probably pretty much the exact same as Discord themselves use (?) - is unnecessary detailed for our use case; why would we care exactly which characters and how many that the usernames consist of?

So I think it's best to simply remove the Discord username validation altogether, including [the `validators` argument](https://github.com/MAKENTNU/web/blob/70270b1eb6e69f2d2dfd99e84d179f17a6483e34/src/internal/models.py#L60) 🙂